examples/lambdarank/README.md

```diff
@@ -21,6 +21,10 @@ You should finish training first.
 
 Run the following command in this folder:
 
+```bash
+../../lightgbm config=predict.conf
 ```
-"./lightgbm" config=predict.conf
-```
+
+Data Format
+-----------
+To learn more about the query format used in this example, check out the [query data format](https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst#query-data).
```

Comment:
Use link to docs site `https://lightgbm.readthedocs.io/`, as you done for installation instructions.

Reply:
Hmmm, but this format is not documented on the documentation website. Unless I missed something? I only found https://lightgbm.readthedocs.io/en/latest/Quick-Start.html#weight-and-query-group-data, but this section doesn't actually explain the format and it links to https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst#query-data.

Is there a better section on the read the docs website? Or, is there a way I could update the documentation website? (I don't know if I can from a PR?)

Reply:
https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst#query-data == https://lightgbm.readthedocs.io/en/latest/Parameters.html#query-data

Reply:
Ok, got it! :D